From e228c5b8f376163c7a5b31c28712cdc8f5863cc0 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Thu, 28 Sep 2017 16:13:46 -0700
Subject: [PATCH] DAPI-345 filter out CNs not matching platform requirements
 for VMs that mount volumes Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
 Approved by: Josh Wilsdon <jwilsdon@joyent.com>

---
 tools/setup/setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/setup/setup.sh b/tools/setup/setup.sh
index f74309b..ba3b4ed 100644
--- a/tools/setup/setup.sh
+++ b/tools/setup/setup.sh
@@ -295,6 +295,7 @@ sdcadm post-setup volapi
 
 upgrade_core_service_to_latest_branch_image "sdc" "tritonnfs"
 upgrade_core_service_to_latest_branch_image "workflow" "tritonnfs"
+upgrade_core_service_to_latest_branch_image "cnapi" "tritonnfs"
 upgrade_core_service_to_latest_branch_image "vmapi" "tritonnfs"
 upgrade_core_service_to_latest_branch_image "docker" "tritonnfs"
 upgrade_core_service_to_latest_branch_image "cloudapi" "tritonnfs"
-- 
2.21.0

